data/json/furniture_and_terrain/furniture-tools.json

```diff
@@ -470,7 +470,7 @@
     "id": "f_arc_furnace",
     "name": "electric arc furnace",
     "looks_like": "f_machinery_heavy",
-    "description": "Not the kind of furnace you'd heat your house with, this is a device for heating things to extreme temperatures as part of industrial fabrication processes.",
+    "description": "Not the kind of furnace you'd heat your house with, this is a device for heating things to extreme temperatures as part of industrial fabrication processes. Can be used in recipes instead of forge, when supplied with power from UPS.",
```

Comment:
```suggestion
    "description": "Not the kind of furnace you'd heat your house with, this is a device for heating things to extreme temperatures as part of industrial fabrication processes.  Can be used in recipes instead of forge, when supplied with power from UPS.",
```